platform/javascript/engine.js

```diff
@@ -1,3 +1,5 @@
+		// The below is concatenated with generated code, and acts as the end of a
```

Comment:
"The below" sounds strange to me, maybe "the following"? Or "the code below"?

Also see https://english.stackexchange.com/questions/172709/the-above-is-correct-the-below-is-not

Reply:
Good point - changed it to "The following". Thanks!